[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"jspi-boot","plugin":"dideo","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.k6V7vjBgjX"}
,
{"message":"Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:05 UTC] PHP Deprecated:  Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:05 UTC] PHP Warning:  Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:05 UTC] PHP Warning:  Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:06 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19\nStack trace:\n#0 /wordpress/wp-settings.php(545): include_once()\n#1 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#2 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#3 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 19","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:13 UTC] PHP Deprecated:  Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:13 UTC] PHP Warning:  Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:13 UTC] PHP Warning:  Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:14 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19\nStack trace:\n#0 /wordpress/wp-settings.php(545): include_once()\n#1 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#2 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#3 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 19","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:21 UTC] PHP Deprecated:  Function get_currentuserinfo is <strong>deprecated</strong> since version 4.5.0! Use wp_get_current_user() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:21 UTC] PHP Warning:  Undefined variable $userdata in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:21 UTC] PHP Warning:  Attempt to read property \"wp_user_level\" on null in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 20","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:22 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_currentuserinfo() in /wordpress/wp-content/plugins/drop-zone/drop-zone.php:19\nStack trace:\n#0 /wordpress/wp-settings.php(545): include_once()\n#1 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#2 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#3 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/drop-zone/drop-zone.php on line 19","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:47:57 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:48:05 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:48:12 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:49 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:50 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dideo/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"dideo","details":"[18-Apr-2025 21:50:59 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"dideo\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/dideo.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/dideo.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/dideo.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/dideo.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n","log":"/logs/plugins/d/dideo/error.json"}
]
